airbyte-integrations/connectors/source-facebook-marketing-api-singer/source_facebook_marketing_api_singer/source.py

```diff
@@ -33,15 +35,18 @@ class SourceFacebookMarketingApiSinger(SingerSource):
     def __init__(self):
         super().__init__()
 
-    def transform_config(self, raw_config):
-        # todo (cgardens) - this is supposed to be handled in the ui and the api but neither of them are able to handle it right now. issue: https://github.com/airbytehq/airbyte/issues/892
-        return {
+    def transform_config(self, config_path: str, raw_config: json):
+        config = {
             "start_date": raw_config["start_date"],
             "account_id": raw_config["account_id"],
             "access_token": raw_config["access_token"],
             # tap-singer expects a string not a boolean
             "include_deleted": str(raw_config.get("include_deleted", True)),
         }
+        return ConfigContainer(
```

Comment:
see above in `base_singer/source.py`. it is not obvious to me why this interface needs to change. we really don't want a developer to have to know about ConfigContainer at all. can we push it back inside singer source and handle it in there?